configuration/cdap-site.xml

```diff
@@ -617,7 +746,7 @@
     <name>metadata.updates.kafka.topic</name>
     <value>cdap-metadata-updates</value>
     <description>
-      Kafka topic to which metadata update notifications are published
+      CDAP Kafka service topic to which metadata update notifications are published
```

Comment:
Same as above, remove the leading ``CDAP`` since this can be an external Kafka instance